lms/djangoapps/courseware/views/views.py

```diff
@@ -579,7 +579,7 @@ def course_about(request, course_id):
         professional_mode = None
         ecomm_service = EcommerceService()
         is_professional_mode = CourseMode.PROFESSIONAL in modes or CourseMode.NO_ID_PROFESSIONAL_MODE in modes
-        if ecomm_service.is_enabled(request.user) and (is_professional_mode):
+        if ecomm_service.config.checkout_on_ecommerce_service and (is_professional_mode):
```

Comment:
I don't like the idea that we are exposing the underlying .config implementation here -- what if we instead changed [EcommerceService.is_enabled](https://github.com/edx/edx-platform/blob/master/lms/djangoapps/commerce/utils.py#L47) to only confirm user.is_active IF the user is not anonymous?